workshops/book-dash/book-dash-mcr-report.md

```diff
@@ -0,0 +1,162 @@
+# The Turing Way | Book Dash - Manchester
+
+
+* Intro to Book Dash MCR slides: https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/book-dash/IntroBookDashMCR.pdf
+* HackMD: http://bit.ly/book-dash-mcr
+
+## The participants
+
+![https://pbs.twimg.com/media/D6y7be5XYAAF0i2?format=jpg&name=900x900](https://pbs.twimg.com/media/D6y7be5XYAAF0i2?format=jpg&name=900x900)
+
+* Rachael Ainsworth [@rainsworth](https://github.com/rainsworth/)
+* Tarek Allam Jr [@tallamjr](https://github.com/tallamjr)
+* Tania Allard [@trallard](https://github.com/trallard)
+* Oliver Clark [@OliJimbo](https://github.com/OliJimbo)
+* Alex Clarke [@informationcake](https://github.com/informationcake/)
+* Jez Cope [@jezcope](https://github.com/jezcope)
+* Joe Fennell [@joe-fennell](https://github.com/joe-fennell)
+* Patricia Herterich [@pherterich](https://github.com/pherterich)
+* Rosie Higman [@rosiehigman](https://github.com/rosiehigman)
+* Will Hulme [@wjchulme](https://github.com/wjchulme) 
+* Greg Kiar [@gkiar](https://github.com/gkiar) 
+* Clare Liggins [@ClareLiggins](https://github.com/ClareLiggins)
+* Javier Moldon [@jmoldon](https://github.com/jmoldon)
+* Beth Montague-Hellen [@AlfAWolf140](https://github.com/AlfAWolf140)
+* Jade Pickering [@jspickering](http://www.github.com/jspickering)
+* Rosti Readioff [@RostiReadioff](https://github.com/RostiReadioff)
+* Kirstie Whitaker [@KirstieJane](https://github.com/KirstieJane/)
+* Yo Yehudi [@yochannah](http://www.github.com/yochannah)
+
+
+## Our report
+
+### What did we do?
+
+Our goal for the book dash was to bring together participants enthusiastic about reproducibility to contribute to and improve the *Turing Way* book during a one day collaborative event. 
+We held a networking event the evening prior to the book dash for participants to get to know each other and promote any projects that they're working on, in order to have an intense 9 to 5 day working on the book.
+There was a great diversity of lightning talks which were really fun:
+* Rachael talked about the women in data meetup group that she organises in Manchester: HER+Data MCR. :woman_technologist:
+* Beth revealed what you can and cannot get away with in roller derby! :women_wrestling:
+* Jez spoke about the tradition of Morris dancing and his experience as a Morris dancer. 🕺
+* Oli wowed us all with a performance of the *Jabberwocky* poem written by Lewis Carroll. 🐉
+* Rosti pitched her Soapbox science plans taking place in Stoke-on-Trent. 🔬
+* Alex described his Music and Machine Learning projects, as well as his students' project on using ML to classify Gender of voices (and the complications and biases involved). 🎶
+* Greg emphasised that a great way to learn more about a sport is through statistics! 🏀
+* Kirstie impressed us with her experiences of dog training. 🐶
+* Joe informed us about a bomb detector made from bees. 🐝
+
```

Comment:
Ok, I'm not going to comment on every lightning talk but I really did love them all so much.

I think the diversity of topics was really important and quite unusual, so worth emphasising the prompt and what we told them in the paragraph above.